src/engine/game/game.lua

```diff
@@ -542,6 +542,26 @@ function Game:getSoulPartyMember()
     return current
 end
 
+function Game:setGameTitle(title)
```

Comment:
maybe its a good idea to change "Game" to "Window" in these functions? ex. `setWindowTitle()`